ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -2160,11 +2160,12 @@ private enum UnityLayers {
 
     // Load the flags.
     Interface.ClearFlags();
-    ConfigNode flags =  GameDatabase.Instance.GetAtMostOneNode(flags_);
-    ConfigNode[] individual_flags = flags.GetNodes();
-    foreach (ConfigNode individual_flag in individual_flags) {
-      foreach (String value in individual_flag.GetValues()) {
-        Interface.SetFlag(individual_flag.name, value);
+    ConfigNode flags =
+        GameDatabase.Instance.GetAtMostOneNode(principia_flags_);
```

Comment:
Maybe
```C#
ConfigNode.ValueList flags = GameDatabase.Instance.GetAtMostOneNode(principia_flags_)?.values
if (flags != null) {
  foreach (ConfigNode.Value flag in flags) {
    // etc.
```
Unless by `individual_flag` we mean "as opposed to flag in a subnode" and have dreams of more-structured flags later on?

Reply:
Done.